sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosCatalog.scala

```diff
@@ -621,20 +697,136 @@ class CosmosCatalog
+        val throughputProperties = throughputPropertiesTuple._1
+        val isSharedThroughput = throughputPropertiesTuple._2
+        val prefix = if (isSharedThroughput) {
+          "Shared."
+        } else {
+          ""
+        }
+        val throughputLastModified = ZonedDateTime
+          .ofInstant(throughputProperties.getTimestamp, ZoneOffset.UTC)
+          .format(DateTimeFormatter.ISO_INSTANT)
+        if (throughputProperties.getAutoscaleMaxThroughput == 0) {
+          s"${prefix}Manual|${throughputProperties.getManualThroughput}|$throughputLastModified"
+        } else {
+          // AutoScale|CurrentRU|MaxRU
+          s"${prefix}AutoScale|${throughputProperties.getManualThroughput}|" +
+            s"${throughputProperties.getAutoscaleMaxThroughput}|" +
+            s"$throughputLastModified"
+        }
+      case None => s"Unknown" // Right now should be serverless  - but because serverless isn't GA
+        // yet keeping the contract vague here
+    }
+
+    val pkDefinitionJson = ModelBridgeInternal
+      .getJsonSerializable(
+        containerProperties.getPartitionKeyDefinition)
+      .toJson
+
+    val tableProperties = new util.HashMap[String, String]()
+    tableProperties.put(
+      CosmosConstants.TableProperties.PartitionKeyDefinition,
+      s"'$pkDefinitionJson'"
+    )
+
+    tableProperties.put(
+      CosmosConstants.TableProperties.PartitionCountSnapshot,
+      s"'${feedRanges.size.toString}'"
+    )
+
+    tableProperties.put(
+      CosmosConstants.TableProperties.ProvisionedThroughputSnapshot,
+      s"'$provisionedThroughputSnapshot'"
+    )
+    tableProperties.put(
+      CosmosConstants.TableProperties.LastModified,
+      s"'$lastModifiedSnapshot'"
+    )
+    tableProperties.put(
+      CosmosConstants.TableProperties.DefaultTtlInSecondsSnapshot,
+      s"'$defaultTimeToLiveInSecondsSnapshot'"
+    )
+    tableProperties.put(
+      CosmosConstants.TableProperties.AnalyticalStorageTtlInSecondsSnapshot,
+      s"'$analyticalStorageTimeToLiveInSecondsSnapshot'"
+    )
+    tableProperties.put(
+      CosmosConstants.TableProperties.IndexingPolicySnapshot,
+      s"'$indexingPolicySnapshotJson'"
+    )
+    tableProperties.put(
+      CosmosConstants.TableProperties.ChangeFeedPolicySnapshot,
+      s"'$changeFeedPolicyPolicySnapshotJson'"
+    )
+
+    tableProperties
+  }
+  // scalastyle:on cyclomatic.complexity
+  // scalastyle:on method.length
+
   private object CosmosContainerProperties {
     val OnlySystemPropertiesIndexingPolicyName: String = "OnlySystemProperties"
     val AllPropertiesIndexingPolicyName: String = "AllProperties"
 
     private val partitionKeyPath = "partitionKeyPath"
+    private val partitionKeyVersion = "partitionKeyVersion"
     private val indexingPolicy = "indexingPolicy"
     private val defaultTtlPropertyName = "defaultTtlInSeconds"
+    private val analyticalStorageTtlPropertyName = "analyticalStorageTtlInSeconds"
```

Comment:
I think the feature is called "analytical store ttl" not "analytical storage ttl" see this:
https://docs.microsoft.com/en-us/azure/cosmos-db/analytical-store-introduction#analytical-ttl

in general the docs refer to the feature as "analytical store"

Reply:
Good catch - thanks. I was mislead by the naming in the Container REST contract
https://github.com/Azure/azure-sdk-for-java/blob/01ad248c46763435bbce35750e23297a46070f2d/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Constants.java#L161

Reply:
Fixed in next iteration